fuel-core/src/database/metadata.rs

```diff
@@ -15,6 +15,7 @@ pub(crate) const FINALIZED_DA_HEIGHT_KEY: &[u8] = b"finalized_da_height";
 pub(crate) const VALIDATORS_DA_HEIGHT_KEY: &[u8] = b"current_validator_set";
 pub(crate) const LAST_COMMITTED_FINALIZED_BLOCK_HEIGHT_KEY: &[u8] =
```

Comment:
Do we still need these constants and the code associated with them?

Reply:
We would have a git history of the block commitment logic, so it's probably better to clean up logic we aren't sure if we'll need. There's a chance we'll come up with a way to never require the relayer to post anything to Ethereum.